modules/FindMagnumIntegration.cmake

```diff
@@ -0,0 +1,309 @@
+#.rst:
```

Comment:
No way to avoid copying these two files into our repo? We can't use the ones in the magnum repo directly somehow?